src/test/java/io/vertx/test/core/DNSTest.java

```diff
@@ -55,6 +55,8 @@ public void testIllegalArguments() throws Exception {
     assertNullPointerException(() -> dns.resolvePTR(null, ar -> {}));
     assertNullPointerException(() -> dns.resolveNS(null, ar -> {}));
     assertNullPointerException(() -> dns.resolveSRV(null, ar -> {}));
+
+    dnsServer.stop();
```

Comment:
I started off doing that, but it seemed weird to clean up dnsServer inside a tearDown method when it is constructed inside the test.  Symmetry and all.  There is also a test which doesn't use it, so the tearDown would have to be intelligent.